docs/intentReaderHandlers/ClearEventHandler.md

```diff
@@ -0,0 +1,139 @@
+# ClearEventHandler 
+
+## Overview
+
+ClearEventHandler is invoked when the task specified in the intent has the value "clearEventHandler". This handler is used to deregister an event that has been registered already.
+
+It performs the following actions :
+1. Parse the input message received to get the event.
+2. Deregister the single event passed in the params field. 
+3. Save the response/error.
+4. Format the result and send the response back to IntentReader.
+
+## Usage
+* This handler is invoked to clear a single event at a time.
+
+```json
+    {
+        "action": "search",
+        "data": {
+            "query": {
+                "query": {
+                    "task": "clearEventHandler",
+                    "params": {
+                        "event": "<eventName>"
+                    },
+                    "action": "NA",
+                    "appType": "<appType>"
+                }
+            }
+        },
+        "context": {
+            "source": "device"
+        }
+    }
+```
+
+### Parameters
+
+| Key                   | Description                                                                           | Required? |
+|-----------------------|---------------------------------------------------------------------------------------|-----------|
+| task                  | "clearEventHandler"- Its a static value and should not be changed for this handler    | Y         |
+| params                | Required event params for the intent. Here, "event" is the mandatory parameter        | Y         |
+| appType               | Corresponding intent is launching on which app                                        | Y         |
+
+
+## Examples
```

Comment:
Similar to `CallMethodHandler` wondering if we can add some visual distinction between examples to make it clear to readers where one example starts and ends.

Additionally the Requests/responses here are mostly not in code blocks. I think we should code block all the requests/responses for readability.

Reply:
added changes